### PR TITLE
Add compat data for apis MouseScrollEvent and MouseWheelEvent

### PR DESCRIPTION
**Pull request by luciancic**
[MouseScrollEvent](https://developer.mozilla.org/docs/Web/API/MouseScrollEvent)
[MouseWheelEvent](https://developer.mozilla.org/docs/Web/API/MouseWheelEvent)

### CODE REVIEW

File: api/MouseScrollEvent.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "3.5"
+          },
+          "firefox_android": {
+            "version_added": "4"
```
Comment (connorshea) [inline api/MouseScrollEvent.json]: On the MDN page, Firefox Mobile says "1", is Firefox Mobile different from Firefox Android?

https://developer.mozilla.org/en-US/docs/Web/API/MouseScrollEvent#Browser_compatibility

File: api/MouseWheelEvent.json [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "wheelDeltaX": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MouseWheelEvent/wheelDeltaX",
```
Comment (connorshea) [inline api/MouseWheelEvent.json]: Are there guidelines on invalid MDN URLs? I doubt a page would ever be created for this.

File: api/MouseWheelEvent.json [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": false,
+          "deprecated": true
+        }
+      },
+      "wheelDelta": {
```
Comment (connorshea) [inline api/MouseWheelEvent.json]: The MDN compat table doesn't mention this property (the page _does_ mention this property) but it can probably be assumed it's got the same support as `wheelDeltaY` and `wheelDeltaX`, I guess?

File: api/MouseWheelEvent.json (hunk 2 above)
Comment (lucian95) [reply api/MouseWheelEvent.json]: I always put them and they automatically get rendered in red; [see here for an example](https://developer.mozilla.org/docs/Web/API/TransitionEvent#Browser_compatibility). 
The way I see it, there are 2 advantages: 
1. The reader can quickly see that there is no page (otherwise it's ambiguous).
2. When the page is added you don't need to make a pull request to add the url.

File: api/MouseScrollEvent.json (hunk 1 above)
Comment (lucian95) [reply api/MouseScrollEvent.json]: Excellent question! Read about why that is so [here](https://github.com/mdn/browser-compat-data/pull/1715).
TL;DR when you see `firefox_android` of 1 to 3, write 4; and when 11 to 13, write 14.
